waterlog/src/dummy.js

```diff
@@ -0,0 +1,82 @@
+export const segmentEvents = [
+    {
+        id: 1,
+        SegmentID: "Segment 1",
```

Comment:
keep casing consistent on variable naming, i.e. either have them start with lower case or have them all start with upper case, just keep it the same

Reply:
the file was deleted, it was for testing